python/cuml/test/test_tsvd.py

```diff
@@ -66,14 +73,18 @@ def test_tsvd_fit_transform(datatype, input_type):
         Xcutsvd = cutsvd.fit_transform(X)
 
     assert array_equal(Xcutsvd, Xsktsvd, 1e-3, with_sign=True)
+    if stream is not None:
```

Comment:
Don't we need to sync in any case? Also is the sync not needed before line 75?

Reply:
Done